packages/core/src/LottieView.tsx

```diff
@@ -83,6 +86,40 @@ export class LottieView extends React.PureComponent<Props, {}> {
     }
   }
 
+  _parsePossibleSources():
```

Comment:
I love this function, but considering what it does and how complicated it can be for someone who doesn't know what it does under the hood, i feel like writing a test will make more sense. This way they won't get confused if they ever manage to figure out a way to not serialize the data and want to modify this function